cmake/cmake_uninstall.cmake.in

```diff
@@ -10,7 +10,7 @@ foreach(file ${files})
 	message(STATUS "Uninstalling $ENV{DESTDIR}${file}")
 	if(IS_SYMLINK "$ENV{DESTDIR}${file}" OR EXISTS "$ENV{DESTDIR}${file}")
 		execute_process(
-			COMMAND @CMAKE_COMMAND@ -E remove $ENV{DESTDIR}${file}
+			COMMAND rm -f $ENV{DESTDIR}${file}
```

Comment:
this will work on windows? I mean in cmd there is no rm

Reply:
In PowerShell it works well. Just checked locally that the `-f` option is unrecognized, so I've removed it.

Reply:
we have to make it work under both PowerShell and CMD

Reply:
`rm` is not a fully portable command:
```
[Windows cmd.exe] C:\Users\user>rm
'rm' is not recognized as an internal or external command,
operable program or batch file.
```
see my comment below

Reply:
Updated with a proposed CMake FILE(REMOVE) command. This should work regardless of the shell used as this is handled by CMake now.